.env.development

```diff
@@ -13,8 +13,8 @@ REACT_APP_GA_TRACKING_ID=''
 REACT_APP_STRIPE_PUBLISHABLE_KEY=pk_test_5AByIibLOhR6WHL3Mwnmel3P00zm0pIDrD
 
 # Square publishable keys
-REACT_APP_SQUARE_APPLICATION_ID=sandbox-sq0idb-aUT15n3qbkvAUY2OXQt_fg
-REACT_APP_SQUARE_LOCATION_ID=E4R1NCMHG7B2Y
+REACT_APP_SQUARE_APPLICATION_ID=sandbox-sq0idb-IDT6EwHSQXv2E91PLPaN5Q
+REACT_APP_SQUARE_LOCATION_ID=XPFZV5MHG689R
```

Comment:
Are we supposed to be committing this?

Reply:
Woops -- left that in from testing, thanks for the catch! Reverted!